spec/search_spec.rb

```diff
@@ -71,6 +71,24 @@ module Search
                     QueryGraph::Graph.from_path([user.id_field]))
         ]
       end
+
+      it 'total cost increases when new update is added' do
+        query = Statement.parse 'SELECT User.UserId FROM User WHERE ' \
+                         'User.City = ? ORDER BY User.Username', workload.model
```

Comment:
This is minor, but the quote on the second line should be aligned with the quote on the first line.